squad/plugins/linux_log_parser.py

```diff
@@ -106,11 +111,27 @@ def __create_tests(self, testrun, suite, test_name, lines):
                 environment=testrun.environment,
             )
 
+    def __remove_numbers_and_time(self, snippet):
+        without_numbers = re.sub(r"(0x[a-f0-9]+|[<\[][0-9a-f]+?[>\]]|\d+)", "", snippet)
+        without_time = re.sub(r"^\[[^\]]+\]", "", without_numbers)
+
+        return without_time
+
+    def __create_name(self, snippet, regex=None):
+        matches = None
+        if regex:
+            matches = regex.findall(snippet)
+        if not matches:
+            return None
+        snippet = matches[0]
+        without_numbers_and_time = self.__remove_numbers_and_time(snippet)
+
+        return slugify(without_numbers_and_time)[:191]
```

Comment:
Why `191` here?

Reply:
Updated with a code comment - max name length is 256 for the name in SQUAD and we have 64 character SHA + a "-" to accommodate